lib/datasets/volume.rb

```diff
@@ -32,5 +32,9 @@ def to_s
       "#{namespace}.#{id} #{right} #{access_profile}"
     end
 
+    def hash
```

Comment:
Needs to be defined for `Set` to work properly. Might be a comment in the spec for this?